docs/conf.py

```diff
@@ -113,7 +114,7 @@
 # -- InterSphinx --------------------------------------------------------------
 
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3.8/", None),
+    "python": ("https://docs.python.org/3/", None),
```

Comment:
This belongs to a separated PR or, at least, to a separated commit within this PR.